Makefile

```diff
@@ -21,7 +21,7 @@ GOVERSION ?= 1.17
 LINUX_ARCH ?= amd64
 WINDOWS_DISTROS ?=
 
-WINDOWS_BASE_IMAGES=$(addprefix mcr.microsoft.com/windows/servercore:,$(WINDOWS_DISTROS))
+WINDOWS_BASE_IMAGES=$(addprefix mcr.microsoft.com/windows/nanoserver:,$(WINDOWS_DISTROS))
```

Comment:
I'm not sure if this base image would work, I haven't tested it

Reply:
yes, it's fixed, all linux and windows images are built succesfully now: https://console.cloud.google.com/gcr/images/k8s-staging-sig-storage/global/local-volume-provisioner

Reply:
btw, we don't want to use `servercore`, that's 2.6GB, it's horrible